From 7a0585105a84f49f28ad760da1091379e1f50dce Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Tue, 23 Apr 2019 12:13:42 +0300
Subject: [PATCH 1098/1200] doc: a7k-a8k: fix RGMIIX naming

Change RGMII0 to RGMII1 and RGMII1 to RGMII2.
In RGMIIX - X is Packet processor port connected to RGMII.

Change-Id: I0c48c88aad037fe3167868506ebd74bf141172fb
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/7971
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Kostya Porotchkin <Kostya.Porotchkin@cavium.com>
---
 doc/mvebu/a7k-a8k/armada70x0-db-setup.txt | 16 +++++------
 doc/mvebu/a7k-a8k/armada80x0-db-setup.txt | 34 +++++++++++------------
 2 files changed, 25 insertions(+), 25 deletions(-)

diff --git a/doc/mvebu/a7k-a8k/armada70x0-db-setup.txt b/doc/mvebu/a7k-a8k/armada70x0-db-setup.txt
index 1ec3722f68..fc306fa5f3 100644
--- a/doc/mvebu/a7k-a8k/armada70x0-db-setup.txt
+++ b/doc/mvebu/a7k-a8k/armada70x0-db-setup.txt
@@ -140,9 +140,9 @@ Multi-purpose pin configurations
 +---------------+---------------+---------------+---------------+---------------+---------------+
 | NAND		| N/C		| N/C		| N/C		| [15-27]	| N/C		|
 +---------------+---------------+---------------+---------------+---------------+---------------+
-| RGMII0	| [0-11]	| [0-11]	| N/C		| N/C		| [0-11]	|
+| RGMII1	| [0-11]	| [0-11]	| N/C		| N/C		| [0-11]	|
 +---------------+---------------+---------------+---------------+---------------+---------------+
-| RGMII1	| N/C		| [44-55]	| [44-55]	| [44-55]	| N/C		|
+| RGMII2	| N/C		| [44-55]	| [44-55]	| [44-55]	| N/C		|
 +---------------+---------------+---------------+---------------+---------------+---------------+
 | CP-UART0	| [29-30]	| [29-30]	| [29-30]	| [29-30]	| [29-30]	|
 +---------------+---------------+---------------+---------------+---------------+---------------+
@@ -179,7 +179,7 @@ Setup 1 (A):
 +---------------+-------+---------------+---------------+-------------------------------+
 | eth1		| CP0	| eth1		| 2		| SGMII1			|
 +---------------+-------+---------------+---------------+-------------------------------+
-| eth2		| CP0	| eth2		| 3		| RGMII1			|
+| eth2		| CP0	| eth2		| 3		| RGMII2			|
 +---------------+-------+---------------+---------------+-------------------------------+
 
 Setup 2 (B):
@@ -187,9 +187,9 @@ Setup 2 (B):
 +---------------+-------+---------------+---------------+-------------------------------+
 | Interface	| CP#   | PPv2 Port     | GMAC Port     | Board Interface               |
 +===============+=======+===============+===============+===============================+
-| eth0		| CP0   | eth0          | 2             | RGMII0                        |
+| eth0		| CP0   | eth0          | 2             | RGMII1                        |
 +---------------+-------+---------------+---------------+-------------------------------+
-| eth1		| CP0   | eth1          | 3             | RGMII1                        |
+| eth1		| CP0   | eth1          | 3             | RGMII2                        |
 +---------------+-------+---------------+---------------+-------------------------------+
 
 Setup 3 (C):
@@ -199,7 +199,7 @@ Setup 3 (C):
 +===============+=======+===============+===============+===============================+
 | eth0		| CP0   | eth0          | 0             | 10G (Lane 2 connected to SFP) |
 +---------------+-------+---------------+---------------+-------------------------------+
-| eth1		| CP0   | eth1          | 3             | RGMII1                        |
+| eth1		| CP0   | eth1          | 3             | RGMII2                        |
 +---------------+-------+---------------+---------------+-------------------------------+
 
 Setup 4 (D):
@@ -209,7 +209,7 @@ Setup 4 (D):
 +===============+=======+===============+===============+===============================+
 | eth0		| CP0   | eth0          | 0             | 10G (Lane 2 connected to SFP) |
 +---------------+-------+---------------+---------------+-------------------------------+
-| eth1		| CP0   | eth1          | 3             | RGMII1                        |
+| eth1		| CP0   | eth1          | 3             | RGMII2                        |
 +---------------+-------+---------------+---------------+-------------------------------+
 
 Setup 5 (E)
@@ -219,7 +219,7 @@ Setup 5 (E)
 +===============+=======+===============+===============+===============================+
 | eth0		| CP0   | eth0          | 0             | SGMII0 (1G)                   |
 +---------------+-------+---------------+---------------+-------------------------------+
-| eth1		| CP0   | eth1          | 2             | RGMII0                        |
+| eth1		| CP0   | eth1          | 2             | RGMII1                        |
 +---------------+-------+---------------+---------------+-------------------------------+
 
 Notes:
diff --git a/doc/mvebu/a7k-a8k/armada80x0-db-setup.txt b/doc/mvebu/a7k-a8k/armada80x0-db-setup.txt
index 1b4992557f..38a5f50902 100644
--- a/doc/mvebu/a7k-a8k/armada80x0-db-setup.txt
+++ b/doc/mvebu/a7k-a8k/armada80x0-db-setup.txt
@@ -160,7 +160,7 @@ Multi-purpose pin configurations
 +---------------+---------------+---------------+---------------+---------------+---------------+---------------+---------------+---------------+
 |   CP0-SATA0	| [41]		| [41]		| [41]		| [41]		| [41]		| [41]		| [41]		| [41]		|
 +---------------+---------------+---------------+---------------+---------------+---------------+---------------+---------------+---------------+
-|   CP0-RGMII1	| [44-55]	| [44-55]	| [44-55]	| [44-55]	| N/C		| [44-55]	| [44-55]	| [44-55]	|
+|   CP0-RGMII2	| [44-55]	| [44-55]	| [44-55]	| [44-55]	| N/C		| [44-55]	| [44-55]	| [44-55]	|
 +---------------+---------------+---------------+---------------+---------------+---------------+---------------+---------------+---------------+
 |   CP0-SD	| [56-62]	| [56-62]	| [56-62]	| [56-62]	| [56-62]	| [56-62]	| [56-62]	| [56-62]	|
 +---------------+---------------+---------------+---------------+---------------+---------------+---------------+---------------+---------------+
@@ -181,7 +181,7 @@ GPIOS:
 +---------------+---------------+---------------+---------------+---------------+---------------+---------------+---------------+---------------+
 | CP1 pin	| 1: A		| 2: B		| 3: C		| 4: D		| 5: E		| 6: F		| 7: G		| 8: H		|
 +===============+===============+===============+===============+===============+===============+===============+===============+===============+
-|   CP1-RGMII0	| [0-11]	| [0-11]	| N/C		| N/C		| [0-11]	| [0-11]	| [0-11]	| [0-11]	|
+|   CP1-RGMII1	| [0-11]	| [0-11]	| N/C		| N/C		| [0-11]	| [0-11]	| [0-11]	| [0-11]	|
 +---------------+---------------+---------------+---------------+---------------+---------------+---------------+---------------+---------------+
 |   CP1-SMI	| [27,31]	| [27,31]	| N/C		| N/C		| [27,31]	| [27,31]	| [27,31]	| [27,31]	|
 +---------------+---------------+---------------+---------------+---------------+---------------+---------------+---------------+---------------+
@@ -210,11 +210,11 @@ Setup 1 (A):
 +===============+=======+===============+===============+===============================+
 | eth0		| 0	| 0		| 0		| 10G (Lane 2 connected to SFP)	|
 +---------------+-------+---------------+---------------+-------------------------------+
-| eth1		| 0	| 2		| 3		| RGMII1			|
+| eth1		| 0	| 2		| 3		| RGMII2			|
 +---------------+-------+---------------+---------------+-------------------------------+
 | eth2		| 1	| 0		| 0		| 10G (Lane 2 connected to SFP)	|
 +---------------+-------+---------------+---------------+-------------------------------+
-| eth3		| 1	| 1		| 2		| RGMII0			|
+| eth3		| 1	| 1		| 2		| RGMII1			|
 +---------------+-------+---------------+---------------+-------------------------------+
 
 Setup 2 (B):
@@ -222,11 +222,11 @@ Setup 2 (B):
 +---------------+-------+---------------+---------------+-------------------------------+
 | Interface	| CP#	| PPv2 Port	| GMAC Port	| Board Interface		|
 +===============+=======+===============+===============+===============================+
-| eth0		| 0	| 2		| 3		| RGMII1			|
+| eth0		| 0	| 2		| 3		| RGMII2			|
 +---------------+-------+---------------+---------------+-------------------------------+
 | eth1		| 1	| 0		| 0		| 10G (Lane 2 connected to SFP)	|
 +---------------+-------+---------------+---------------+-------------------------------+
-| eth2		| 1	| 1		| 2		| RGMII0			|
+| eth2		| 1	| 1		| 2		| RGMII1			|
 +---------------+-------+---------------+---------------+-------------------------------+
 
 Setup 3 (C):
@@ -236,7 +236,7 @@ Setup 3 (C):
 +===============+=======+===============+===============+===============================+
 | eth0		| 0	| 0		| 0		| 10G (Lane 2 connected to SFP)	|
 +---------------+-------+---------------+---------------+-------------------------------+
-| eth1		| 0	| 2		| 3		| RGMII1			|
+| eth1		| 0	| 2		| 3		| RGMII2			|
 +---------------+-------+---------------+---------------+-------------------------------+
 | eth2		| 1	| 0		| 0		| 10G (Lane 2 connected to SFP)	|
 +---------------+-------+---------------+---------------+-------------------------------+
@@ -248,9 +248,9 @@ Setup 4 (D):
 +===============+=======+===============+===============+===============================+
 | eth0		| 0	| 0		| 0		| 10G (Lane 2 connected to SFP)	|
 +---------------+-------+---------------+---------------+-------------------------------+
-| eth1		| 0	| 2		| 3		| RGMII1			|
+| eth1		| 0	| 2		| 3		| RGMII2			|
 +---------------+-------+---------------+---------------+-------------------------------+
-| eth2		| 1	| 0		| 0		| SGMII				|
+| eth2		| 1	| 0		| 0		| SGMII0			|
 +---------------+-------+---------------+---------------+-------------------------------+
 
 Setup 5 (E):
@@ -260,9 +260,9 @@ Setup 5 (E):
 +===============+=======+===============+===============+===============================+
 | eth0		| 0	| 0		| 0		| 10G (Lane 2 connected to SFP)	|
 +---------------+-------+---------------+---------------+-------------------------------+
-| eth1		| 1	| 0		| 0		| 2.5G SGMII			|
+| eth1		| 1	| 0		| 0		| 2.5G SGMII0			|
 +---------------+-------+---------------+---------------+-------------------------------+
-| eth2		| 1	| 1		| 2		| RGMII0			|
+| eth2		| 1	| 1		| 2		| RGMII1			|
 +---------------+-------+---------------+---------------+-------------------------------+
 
 Setup 6 (F):
@@ -272,11 +272,11 @@ Setup 6 (F):
 +===============+=======+===============+===============+===============================================+
 | eth0		| 0	| 0		| 0		| AP (Lane 2 left for 802.3 auto-negotiation)	|
 +---------------+-------+---------------+---------------+-----------------------------------------------+
-| eth1		| 0	| 2		| 3		| RGMII1					|
+| eth1		| 0	| 2		| 3		| RGMII2					|
 +---------------+-------+---------------+---------------+-----------------------------------------------+
 | eth2		| 1	| 0		| 0		| 10G (Lane 2 connected to SFI)			|
 +---------------+-------+---------------+---------------+-----------------------------------------------+
-| eth3		| 1	| 1		| 2		| RGMII0					|
+| eth3		| 1	| 1		| 2		| RGMII1					|
 +---------------+-------+---------------+---------------+-----------------------------------------------+
 
 Setup 7 (G):
@@ -284,9 +284,9 @@ Setup 7 (G):
 +---------------+-------+---------------+---------------+-------------------------------+
 | Interface	| CP#	| PPv2 Port	| GMAC Port	| Board Interface		|
 +===============+=======+===============+===============+===============================+
-| eth0		| 0	| 2		| 3		| RGMII1			|
+| eth0		| 0	| 2		| 3		| RGMII2			|
 +---------------+-------+---------------+---------------+-------------------------------+
-| eth1		| 1	| 1		| 2		| RGMII0			|
+| eth1		| 1	| 1		| 2		| RGMII1			|
 +---------------+-------+---------------+---------------+-------------------------------+
 
 Setup 8 (H):
@@ -296,11 +296,11 @@ Setup 8 (H):
 +===============+=======+===============+===============+===============================+
 | eth0		| 0	| 0		| 0		| 10G (Lane 4 connected to SFP)	|
 +---------------+-------+---------------+---------------+-------------------------------+
-| eth1		| 0	| 2		| 3		| RGMII1			|
+| eth1		| 0	| 2		| 3		| RGMII2			|
 +---------------+-------+---------------+---------------+-------------------------------+
 | eth2		| 1	| 0		| 0		| 10G (Lane 4 connected to SFP)	|
 +---------------+-------+---------------+---------------+-------------------------------+
-| eth3		| 1	| 1		| 2		| RGMII0			|
+| eth3		| 1	| 1		| 2		| RGMII1			|
 +---------------+-------+---------------+---------------+-------------------------------+
 
 Notes:
-- 
2.22.0

